built-in_analyses.js

```diff
@@ -256,7 +287,7 @@ function built_in_Japanese_IM2017(){
 
 //cf. analysis_html_files/abstractMatchAnalysis.html. Japanese rebracketing project, Kalivoda 2019.
 function built_in_Japanese_rebracketing(n){
-  var gen = {obeysExhaustivity: true, requireRecWrapper: true};
+  var gen = {obeysExhaustivity: true, requireRecWrapper: true, rootCategory: "phi"};
```

Comment:
Ah, I see what happened. requireRecWrapper doesn't work on the interface, only on the backend.